scripts/create_changelog.sh

```diff
@@ -1,16 +1,21 @@
 #!/bin/bash
+set -u -e -o pipefail
 
 VERSION=$(grep '^VERSION' setup.py | sed -E "s/.*'([^']*)'/\1/")
 RELEASE_DATE=$(date "+%Y-%m-%d")
 
+# compile a list of already reported PRs
+tempfile="$(mktemp)"
```

Comment:
Better name for the variable or for the file?
I can change it if you have a suggestion.
`reported="$(mktemp)"` ?